wordpress-mobile/WordPress-iOS
# Adapt NoticeView to bigger font sizes

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/10846

**To test:**
Create a site and enter the QuickStart tour and see the message in the related issue ☝️ 

Another option could be mocking:

There's a `Revert fake notice` commit included in this PR, if that commit is reverted in the local environment a notice is seen on blog list page.
 
Update release notes:
